plugins/Trayicon/TrayiconPlugin.py

```diff
@@ -115,9 +115,9 @@ def formatAutorun(self):
         cmd = " ".join(args)
 
         # Dont open browser on autorun
-        cmd = cmd.replace("start.py", "zeronet.py").replace('"--open_browser"', "").replace('"default_browser"', "")
+        cmd = cmd.replace("start.py", "zeronet.py").replace(' "--open_browser"', "").replace(' "default_browser"', "")
```

Comment:
It could be replaced with `cmd = cmd.replace('start.py --open_browser "default_browser"', "zeronet.py")` ?